docs/release-notes.txt

```diff
@@ -8,6 +8,57 @@
  Version 0.13
 **************
 
+Version 0.13.5
+==============
+
+**Release Date:** September 30, 2020
+
+**Improvements**
+
+- Security: Use one TCP port for all incoming connections to the master and use TLS for all connections if configured.
+
+  - **Breaking Change:** The ``http_port`` and ``https_port`` options in the master configuration have been replaced by the single ``port`` option. The ``security.http`` option is no longer accepted; the master can no longer be configured to listen over HTTP and HTTPS simultaneously.
+
+- Security: Support configuring TLS encryption when deploying Determined on Kubernetes. For more details please see :ref:`install-on-kubernetes`.
+
+- Agent: Increase default max agent starting and idle timeouts to 20 minutes and increase max disconnected period from 5 to 10 minutes.
+
+- API: Deprecate Native API. Removed related examples and docs.
+
+- Deployment: Add support for ``det-deploy aws`` in the following new regions: ``ap-northeast-1``, ``eu-central-1``, ``eu-west-1``, ``us-east-2``.
+
+- Documentation: Add extra documentation to and reorganize examples by use case.
+
+- Documentation: Add a ``tf.layers-in-Estimator`` example.
+
+- Kubernetes: Add support for users to specify ``initContainers`` and ``containers`` as part of their custom pod specs. Please see :ref:`custom-pod-specs` for details.
```

Comment:
Can we also add in that we published `v0.2.0` of the Determined Helm chart

Reply:
Added.